src/controller/CHIPDeviceController.cpp

```diff
@@ -400,13 +401,13 @@ void DeviceController::Shutdown()
         // existing sessions too?
         mSystemState->SessionMgr()->ExpireAllSessionsForFabric(mFabricIndex);
 
-        if (mRemoveFromFabricTableOnShutdown)
+        if (mDeleteFromFabricTableOnShutdown)
         {
-            FabricTable * fabricTable = mSystemState->Fabrics();
-            if (fabricTable != nullptr)
```

Comment:
We know the SystemState is valid here, so Fabrics() never returns null.

Reply:
Should this be in a code comment then?

Reply:
Added a comment